apps/studio/package.json

```diff
@@ -44,8 +44,10 @@
     "jump": "source .ssh/.env.prod && ssh -L 5433:$DB_HOST:5432 $SSH_USER@$SSH_HOST -i .ssh/studio-prod-bastion.pem",
     "jump:staging": "source .ssh/.env.staging && ssh -L 5433:$DB_HOST:5432 $SSH_USER@$SSH_HOST -i .ssh/studio-staging-bastion.pem",
     "jump:uat": "source .ssh/.env.uat && ssh -L 5433:$DB_HOST:5432 $SSH_USER@$SSH_HOST -i .ssh/studio-uat-bastion.pem",
+    "jump:vapt": "source .ssh/.env.vapt && ssh -L 5433:$DB_HOST:5432 $SSH_USER@$SSH_HOST -i .ssh/studio-vapt-bastion.pem",
```

Comment:
double check - this pem key is already in 1pw right?

Reply:
@seaerchin yeap!

Reply:
migration done too